getting_started/configuration_file.md

```diff
@@ -6,22 +6,110 @@ compiler, formatter and linter.
 The configuration file supports `.json` and `.jsonc` extensions.
 [Since v1.18](https://deno.com/blog/v1.18#auto-discovery-of-the-config-file),
 Deno will automatically detect `deno.json` or `deno.jsonc` configuration file if
-it's in your current working directory (or parent directories). To manually tell
-Deno to use a specific configuration file pass `--config path/to/file.json`
-flag.
-
-> ⚠️ Starting with Deno v1.22 you can disable automatic detection of the
-> configuration file, by passing `--no-config`.
-
-Note that using a configuration file is not required now, and will not be
-required in the future. Deno still works best with the default options and no
-configuration file. All options specified in the configuration file can also be
-set using command line flags (for example `--options-use-tabs` for `deno fmt`).
-Using the configuration file should be considered an "as needed" feature, not
-something every user should be reaching to as the first thing when setting up a
-project.
-
-## Example
+it's in your current working directory or parent directories. (`--config` flag
+can be used to specify a different configuration file.)
+
+> ⚠️ Before Deno v1.23 you can needed to supply an explicit `--config` flag.
+
+## `imports` and `scopes`
+
+The `deno.json` file can act as an
+[import map](https://github.com/WICG/import-maps) for resolving bar specifiers.
+
+```json
+{
+  "imports": {
+    "std/": "https://deno.land/std@0.174.0/"
+  }
+}
+```
+
+Then your script can use the bare specifier `std`:
+
+```js
+import { assertEquals } from "std/testing/assert.ts";
+
+assertEquals(1, 2);
+```
+
+The top-level `deno.json` option `importMap` along with the `--importmap` flag
```

Comment:
Should the precedence be documented? I think the workspace settings can also have an import map definition which, according to some LSP error messages I saw in the imports PR, might take precedence over all others? Or maybe that's only for the LSP?